frontend/pages/packs/EditPackPage/components/PackQueryEditorModal/PackQueryEditorModal.tsx

```diff
@@ -140,13 +145,26 @@ const PackQueryEditorModal = ({
   };
 
   const onFormSubmit = (): void => {
+    setErrorFrequency("");
     const query_id = () => {
       if (editQuery) {
         return editQuery.query_id;
       }
       return selectedQuery?.id;
     };
 
+    const f = parseInt(selectedFrequency, 10);
+    if (!f || f < 0) {
+      setErrorFrequency("Frequency must be an integer greater than zero");
+      return;
+    }
+    if (f > MAX_OSQUERY_SCHEDULED_QUERY_INTERVAL) {
+      setErrorFrequency(
+        "Frequency must be an integer that does not exceed 604,800 (i.e. 7 days)"
+      );
+      return;
+    }
+
```

Comment:
Would you mind changing the variable name of `f` to something more descriptive like `frequency`? It's pretty obvious here, but sometimes when folks shorten variable names it's less obvious so I'd like to enforce a rule of always spelling out the word.

Reply:
No problem! I tend to like the more descriptive naming too although Go favors short variable names for short-lived variables. I'm always caught between idioms lol 😅

Reply:
Thanks! I noticed the Go side prefers shorter variable names, while TS prefers longer variable names. Must be confusing switching between the two!

Reply:
Keeps me on my toes for sure!